packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -3607,4 +3608,147 @@ describe("Teal Opcodes", function () {
       );
     });
   });
+
+  describe("GetByte", () => {
+    let stack: Stack<StackElem>;
+    this.beforeEach(() => { stack = new Stack<StackElem>(); });
+
+    it("should get correct bytes from stack", () => {
+      const op = new GetByte([], 0);
+      stack.push(new Uint8Array([8, 2, 1, 9])); // target
+      stack.push(0n); // index
+
+      op.execute(stack);
+      assert.equal(stack.pop(), 8n);
+
+      stack.push(new Uint8Array([8, 2, 1, 9])); // target
+      stack.push(3n); // index
+
+      op.execute(stack);
+      assert.equal(stack.pop(), 9n);
+
+      stack.push(new Uint8Array([1, 2, 3, 4, 5])); // target
+      stack.push(2n); // index
+
+      op.execute(stack);
+      assert.equal(stack.pop(), 3n);
+    });
+
+    it("should panic if target is not bytes", () => {
+      const op = new GetByte([], 0);
+      stack.push(10n); // target
+      stack.push(0n); // index
+
+      expectRuntimeError(
+        () => op.execute(stack),
+        RUNTIME_ERRORS.TEAL.INVALID_TYPE
+      );
+    });
+
+    it("should panic if index is not uint", () => {
```

Comment:
`unit64` or `uint8`?

Reply:
this test is for byte cheking `index should not be bytes`